### DISCUSSION TITLE
Switching to Gmail?

### DISCUSSION THREAD

**Discussion by RobertAKARobin** [Geneticure/geneticure.com#43] [General]
@scottsnyder01 @snydreric @Carl362 

Thoughts on switching from Outlook to Gmail?

# Advantages:

- Gmail comes with Google Chat built-in, so we wouldn't need Slack anymore and could more easily chat with each other
- Gmail comes with Google Meet built-in, so we wouldn't need Uberconference anymore
- Gmail comes with Google Drive built-in, which is considerably better than OneDrive. A bunch of our stuff is in Drive already, anyway
- We wouldn't need to host our own email server, as we currently are with Rackspace
- I don't think we'd need Rackspace at all, actually

Cost: $6/user/month: https://workspace.google.com/pricing.html

# Disadvantages:

- None, other than that we're currently using Outlook.

# How it would work

Gmail has a migration service that should copy over all of our old emails, so it should be pretty seamless. Then I just change a few DNS records and away we go.

Google did not pay me to make this post. :)

**Comment by RobertAKARobin** [comment]
More advantages:

- Pretty sure we all have personal Gmail accounts anyway
- Google Calendar > Outlook Calendar, IMO

**Comment by scottsnyder01** [comment]
I’m in theory on board with this but would need to see how the mobile experience is – I need to have email/calendar in one application to easily schedule hundreds of meetings per yer. I know apple has them separate which is dumb.

Also likely advantages to collaborate on google sheets, etc..which I think is no longer free.





From: "Robin (Robert) Thomas" ***@***.***>
Reply-To: "Geneticure/geneticure.com" ***@***.***>
Date: Friday, September 30, 2022 at 10:07 AM
To: "Geneticure/geneticure.com" ***@***.***>
Cc: Scott Snyder ***@***.***>, Mention ***@***.***>
Subject: [Geneticure] Switching to Gmail? (Discussion #43)


@scottsnyder01<https://github.com/scottsnyder01> @snydreric<https://github.com/snydreric> @Carl362<https://github.com/Carl362>

Thoughts on switching from Outlook to Gmail?

Advantages:

  *   Gmail comes with Google Chat built-in, so we wouldn't need Slack anymore and could more easily chat with each other
  *   Gmail comes with Google Meet built-in, so we wouldn't need Uberconference anymore
  *   Gmail comes with Google Drive built-in, which is considerably better than OneDrive. A bunch of our stuff is in Drive already, anyway
  *   We wouldn't need to host our own email server, as we currently are with Rackspace
  *   I don't think we'd need Rackspace at all, actually

Cost: $6/user/month: https://workspace.google.com/pricing.html

Disadvantages:

  *   None, other than that we're currently using Outlook.

How it would work

Gmail has a migration service that should copy over all of our old emails, so it should be pretty seamless. Then I just change a few DNS records and away we go.

Google did not pay me to make this post. :)

**Comment by RobertAKARobin** [comment]
@scottsnyder01 Sheets/Docs/etc are all included with Drive, so no extra cost for those.

Gmail, Chat, and Meet are all sort-of in the same application. Not sure about Calendar — you can create an event from an email on desktop, but not on mobile for some reason.

Apparently it's because Microsoft has a patent on that? https://qr.ae/pvaKiS 🙄

**Comment by RobertAKARobin** [comment]
@scottsnyder01 Is the mobile app a deal-breaker? There have already been a couple of times when I've wished we could just G-chat each other. FWIW it looks like Google built a tool to sync Gmail with Outlook, so you may not really need to change anything: https://support.google.com/a/users/answer/171710?hl=en

**Comment by snydreric** [comment]
@robin not sure if I am doing this right.... I'm an old man. I was at u of a and u of m for system wide switches to gmail... and it sucked. I have to use my Gmail app on my phone and it also sucks. I hate how it threads emails, etc. Just want to give my unsolicited opinion before a decision is made. 😀

On Oct 6, 2022 8:05 AM, "Robin (Robert) Thomas" ***@***.***> wrote:

@scottsnyder01<https://github.com/scottsnyder01> Is the mobile app a deal-breaker? There have already been a couple of times when I've wished we could just G-chat each other. FWIW it looks like Google built a tool to sync Gmail with Outlook, so you may not really need to change anything: https://support.google.com/a/users/answer/171710?hl=en

**Comment by RobertAKARobin** [comment]
@snydreric Huh, that's surprising to me. I've been using the Gmail app since forever and have never had any issues with it. Just about everywhere I've worked has switched from Microsoft to Google Workspace (General Assembly, MentorMate, Nerdery, my wife's school...)

FWIW threading can be turned off.

**Comment by snydreric** [comment]
Yeah, I've turned threading off but still dont like the look. I'm known for not liking things that other people like (like apple anything), so don't take my word for it but I did want to toss in my old man opinion!



On Oct 6, 2022 8:13 AM, "Robin (Robert) Thomas" ***@***.***> wrote:

@snydreric<https://github.com/snydreric> Huh, that's surprising to me. I've been using the Gmail app since forever and have never had any issues with it. Just about everywhere I've worked has switched from Microsoft to Google Workspace (General Assembly, MentorMate, Nerdery, my wife's school...)

FWIW threading can be turned off.

-

**Comment by RobertAKARobin** [comment]
Well, not a pressing concern. Still, looks like we're paying ~$170 a month for MS on Rackspace (unsure about Onedrive/Slack/Uberconference), and Google would be a fraction of that.

**Comment by scottsnyder01** [comment]
Let me do some asking around to friends and see if I can find anyones company that has switched and how they feel. No bias to your experience but want to hear how it looks from people who have been entrenched in outlook like we all have for many years.

Scott Snyder
952.210.5851
________________________________

**Comment by scottsnyder01** [comment]
I also don’t understand why we are emailing through github when none of us are in there frequently. I guess it’s supposed to be a form of chat but still sends an email to me without all the previous chats so even less functional. Ha.

Scott Snyder
952.210.5851
________________________________

**Comment by RobertAKARobin** [comment]
It keeps all our conversations in GH, so if we ever want to revisit topics
down the road we don't need to wade through emails, and we can have all our
discussions in one place: https://github.com/orgs/Geneticure/discussions/43

Don't think we lose anything by using it.

**Comment by scottsnyder01** [comment]
Ironically all my conversations are in one place and searchable in outlook as well :)

Scott Snyder
952.210.5851
________________________________

**Comment by snydreric** [comment]
We Def can't get rid of one drive. It is part of out clia compliance right now and would be a pain in the ass to switch that over to Google drive (which I mistakenly wiped out once and they did not have the capacity to restore it, which is the qhole purpose of that type of server).

E

On Oct 6, 2022 8:22 AM, "Robin (Robert) Thomas" ***@***.***> wrote:

Well, not a pressing concern. Still, looks like we're paying ~$170 a month for Rackspace (unsure about Onedrive/Slack/Uberconference), and Google would be a fraction of that.

-

**Comment by RobertAKARobin** [comment]
@scottsnyder01 Yes, but when/if we add new employees they won't have access to the emails. From a project management standpoint it's best practice for discussions around decision-making to be documented somewhere other than email. Rather than writing a doc, may as well just use the GH integration to have the emails become the docs. If we feel we're losing something by using GH then we can stop, but otherwise I'd like to keep using it at least for my technical discussions.

@snydreric 👍

**Comment by scottsnyder01** [comment]
makes sense. Hadn’t thought about new employees. Like your growth minded thinking!

Scott Snyder
952.210.5851
________________________________

**Comment by scottsnyder01** [comment]
Onedrive is included in the rackspace cost. Uberconference is $16/month and we can probably kill that and just use zoom. I like that it has a local 612 # for easy dial in vs. zoom for folks that don’t want to click that link (ahhhem, e). If Eric wants to keep it we will, its probably a small but duplicative cost in my opinion

From: "Robin (Robert) Thomas" ***@***.***>
Reply-To: "Geneticure/geneticure.com" ***@***.***>
Date: Thursday, October 6, 2022 at 8:22 AM
To: "Geneticure/geneticure.com" ***@***.***>
Cc: Scott Snyder ***@***.***>, Mention ***@***.***>
Subject: Re: [Geneticure/geneticure.com] Switching to Gmail? (Discussion #43)


Well, not a pressing concern. Still, looks like we're paying ~$170 a month for Rackspace (unsure about Onedrive/Slack/Uberconference), and Google would be a fraction of that.

**Comment by scottsnyder01** [comment]
Not paying for slack..

From: Scott Snyder ***@***.***>
Date: Thursday, October 6, 2022 at 8:49 AM
To: "Geneticure/geneticure.com" ***@***.***>, "Geneticure/geneticure.com" ***@***.***>
Cc: Mention ***@***.***>
Subject: Re: [Geneticure/geneticure.com] Switching to Gmail? (Discussion #43)

Onedrive is included in the rackspace cost. Uberconference is $16/month and we can probably kill that and just use zoom. I like that it has a local 612 # for easy dial in vs. zoom for folks that don’t want to click that link (ahhhem, e). If Eric wants to keep it we will, its probably a small but duplicative cost in my opinion

From: "Robin (Robert) Thomas" ***@***.***>
Reply-To: "Geneticure/geneticure.com" ***@***.***>
Date: Thursday, October 6, 2022 at 8:22 AM
To: "Geneticure/geneticure.com" ***@***.***>
Cc: Scott Snyder ***@***.***>, Mention ***@***.***>
Subject: Re: [Geneticure/geneticure.com] Switching to Gmail? (Discussion #43)


Well, not a pressing concern. Still, looks like we're paying ~$170 a month for Rackspace (unsure about Onedrive/Slack/Uberconference), and Google would be a fraction of that.

**Comment by RobertAKARobin** [comment]
You're Mr Moneybags @scottsnyder01 , $16/mo for a dedicated meeting link sounds acceptable to me, but up to you

**Comment by scottsnyder01** [comment]
It’s really up to eric if he wants to get comfy w zoom we can cancel it.

From: "Robin (Robert) Thomas" ***@***.***>
Reply-To: "Geneticure/geneticure.com" ***@***.***>
Date: Thursday, October 6, 2022 at 8:53 AM
To: "Geneticure/geneticure.com" ***@***.***>
Cc: Scott Snyder ***@***.***>, Mention ***@***.***>
Subject: Re: [Geneticure/geneticure.com] Switching to Gmail? (Discussion #43)


You're Mr Moneybags @scottsnyder01<https://github.com/scottsnyder01> , $16/mo for a dedicated meeting link sounds acceptable to me, but up to you

**Comment by snydreric** [comment]
Haha! I use zoom all the time too. I'm fine with either of those. Don't love Microsoft Teams, but I know Scott is with me on that. We 100% have to keep one drive as is.

On Oct 6, 2022 8:56 AM, scottsnyder01 ***@***.***> wrote:

It's really up to eric if he wants to get comfy w zoom we can cancel it.

From: "Robin (Robert) Thomas" ***@***.***>
Reply-To: "Geneticure/geneticure.com" ***@***.***>
Date: Thursday, October 6, 2022 at 8:53 AM
To: "Geneticure/geneticure.com" ***@***.***>
Cc: Scott Snyder ***@***.***>, Mention ***@***.***>
Subject: Re: [Geneticure/geneticure.com] Switching to Gmail? (Discussion #43)


You're Mr Moneybags @scottsnyder01<https://github.com/scottsnyder01> , $16/mo for a dedicated meeting link sounds acceptable to me, but up to you

-
Reply to this email directly, view it on GitHub<https://github.com/orgs/Geneticure/discussions/43#discussioncomment-3814763>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AIRNRJVP6GQQXLM5QEBG6MDWB3KUFANCNFSM6AAAAAAQZ3QZCU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

-

**Comment by RobertAKARobin** [comment]
Don't think anyone _loves_ MS Teams. The pros of consolidating all of our stuff into one package might outweigh that though
